scripts/docker-integration-tests/dedicated_etcd_embedded_coordinator/m3dbnode.yml

```diff
@@ -50,11 +49,6 @@ db:
   writeNewSeriesBackoffDuration: 2ms
 
   bootstrap:
-    # Intentionally disable peers bootstrapper to ensure it doesn't interfere with test.
```

Comment:
If we want to preserve this behavior, maybe we add an `Enabled` flag to each bootstrap type (e.g. under `peers`, `commitlog`, etc.)

Reply:
Hmm, the tests passed w/o the peers bootstrapper disabled? I think it's prob fine if the docker integration tests passed. 

For integration tests we explicitly configure the bootstrappers we need.

Reply:
Yeah so all the integration tests passed with just the standard ones included (including peers). So I assume it is fine? I'm not sure what the original reason was for excluding them given things are passing.

However, since we removed the `bootstrappers` list from the config, we can no longer select a subset of bootstrappers to run for specific integration tests. So I think if we do end up needing that flexibility, we can just add in an Enabled or Disabled flag to each of the types under the `bootstrap` config section. Does that make sense?

Reply:
If we can do it in integration tests, we can make it a code hook only rather than a config hook if we do end up needing to disable bootstrappers?

Reply:
Seems like the integration tests all work w/ standard bootstrappers enabled so I'd vote we keep everything as-is since it is more comprehensive testing / less special casing

Reply:
+1 to less special casing.